### PR TITLE
Update VCS_Table.bas

### PR DESCRIPTION
**Pull request by PRMills**
Fix missing enum for bit fields (dbBoolean)

### CODE REVIEW

**Comment by PRMills** [comment]
Works on my machine!

**Comment by timabell** [comment]
Thanks for the pull request, did you mean to add the extra two commits to it?

File: MSAccess-VCS/VCS_ImportExport.bas [hunk 1]
```diff
@@ -437,3 +448,8 @@ End Function
 
 
 
+
+
+
+
+
```
Comment (timabell) [inline MSAccess-VCS/VCS_ImportExport.bas]: seems to be a bunch of extra lines added here, I assume this wasn't intentional?

File: MSAccess-VCS/VCS_Table.bas [hunk 2]
```diff
@@ -323,7 +323,7 @@ Public Sub ExportTableData(tbl_name As String, obj_path As String)
     End If
     Set rs = CurrentDb.OpenRecordset(TableExportSql(tbl_name))
     If rs.RecordCount = 0 Then
-        Debug.Print "Error: Table " & tbl_name & "  empty"
+        'why is this an error? Debug.Print "Error: Table " & tbl_name & "  empty"
```
Comment (timabell) [inline MSAccess-VCS/VCS_Table.bas]: I agree that shouldn't be an error. I say just print a warning. Or maybe even take it out entirely. We could just always show the record count. I haven't used the data export much.

**Comment by CrispinF** [comment]
I was looking at this with Paul, and yes we meant to submit all three. Obviously the data error thing doesn't really matter, but the others are needed - or at least they are on our accdbs!

**Comment by timabell** [comment]
cool, thanks. I'll merge it in then.

**Pull request merged**